nbgrader/auth/hubauth.py

```diff
@@ -37,6 +37,11 @@ def _hubapi_address_default(self):
     def _hub_base_url_default(self):
         return 'http://{}:{}'.format(self.hub_address, self.hub_port)
 
+    remap_url = Unicode('/hub/formgrade', config=True, help="""Suffix appened to 
```

Comment:
It seems like this needs to always start with a `/` and end with no `/`. Could there could be some way of validating this, or at least a note about it in the docstring?